### PR TITLE
Bioconductor 3.9

### PR DESCRIPTION
**Pull request by dpryan79**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
I'll merge this manually into bulk, I just want to see let the linting step proceed at least.

**Comment by dpryan79** [comment]
@bioconda/core FYI, this is the bioconductor 3.9 release, which came out on Friday. It now represents approximately 2/3 of all bioconda packages...

**Comment by dpryan79** [comment]
This seems to be running nicely. I'd like to finish going through and manually putting in the system requirements before merging manually into bulk.

File: .circleci/common.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MINICONDA_VER=4.5.4
```
Comment (bgruening) [inline .circleci/common.sh]: Mh, why is this here? 4.5 also seems to be old?

Comment (dpryan79) [reply .circleci/common.sh]: No clue, it was added by someone in `master` at some point and gets overwritten every time.

Comment (dpryan79) [reply .circleci/common.sh]: I just updated the PR to delete that file and add it to .gitignore.

**Comment by dpryan79** [comment]
If there are no other issues (and no warning sirens going off in the linux test) I'll merge this in in ~2 hours.
